.projenrc.js

```diff
@@ -2,13 +2,19 @@ const { GemeenteNijmegenCdkApp } = require('@gemeentenijmegen/modules-projen');
 const project = new GemeenteNijmegenCdkApp({
   cdkVersion: '2.1.0',
   defaultReleaseBranch: 'main',
-  devDeps: ['@gemeentenijmegen/modules-projen'],
+  devDeps: [
+    '@gemeentenijmegen/modules-projen',
+  ],
   name: 'aws-monitoring',
 
   deps: [
     'aws-cdk-lib',
     'constructs',
+    'cdk-nag',
   ],
+  scripts: {
+    'post-upgrade': 'cd src/LogLambda &&  npx npm-check-updates --upgrade --target=minor',
```

Comment:
Is het niet handiger om de projen lambda functie te gebruiken? (gratis package management via projen en typescript) Misschien hier niet heel relevant, maar volgens mij is dat een best handige manier van lambdas maken en package management doen.

Reply:
Daar was ik mee begonnen, maar kreeg ruzie met de build. Nog met een testproject ook geprobeerd, zou iets met de nijmegen-project-versie te maken kunnen hebben. Voor nu geen extra tijd in gestoken om 't uit te zoeken. Ombouwen is kleine moeite als het weer werkt.